website/pages/docs/builders/vmware/vsphere-iso.mdx

```diff
@@ -30,20 +30,33 @@ the items listed here, you will want to look at the general configuration
 references for [HTTP](#http-directory-configuration),
 [Floppy](#floppy-configuration),
 [Boot](#boot-configuration),
-[Driver](#driver-configuration),
 [Hardware](#hardware-configuration),
 [Output](#output-configuration),
 [Run](#run-configuration),
 [Shutdown](#shutdown-configuration),
 [Communicator](#communicator-configuration),
-[Tools](#tools-configuration),
-[vmx](#vmx-configuration),
 [Export](#export-configuration),
 configuration references, which are
 necessary for this build to succeed and can be found further down the page.
 
 @include 'builder/vsphere/iso/Config-not-required.mdx'
 
+### Http directory configuration
+
+@include 'common/HTTPConfig.mdx'
+
+#### Optional:
+
+@include 'common/HTTPConfig-not-required.mdx'
+
+### Floppy configuration
```

Comment:
There's one more section already defined in the doc for the Floppy Configuration [HERE](https://github.com/hashicorp/packer/blob/ab99c36f5f600cbe2c4208cbc99e6078abfb6959/website/pages/docs/builders/vmware/vsphere-iso.mdx#floppy-configuration-1) 
Check it out the PR deployed website version [one section](https://5ed599a564026c00066239b0--packer-www.netlify.app/docs/builders/vmware/vsphere-iso/#floppy-configuration-1) and [the other section](https://5ed599a564026c00066239b0--packer-www.netlify.app/docs/builders/vmware/vsphere-iso/#floppy-configuration)

Reply:
The existing section has one extra config.

Reply:
good catch. I removed the old one since it only had the required fields.

Reply:
what about the [floppy_img_path](https://www.packer.io/docs/builders/vmware/vsphere-iso/#floppy_img_path) information? I can see that it exists still in the code but is being removed from the docs here. Is it correct?

Reply:
life is so confusing. the vsphere-iso doesnt use the common floppy config...added the vsphere-iso specific one back.

Reply:
😆 I agree 100% with you!